src/crystal/system/file.cr

```diff
@@ -40,6 +40,13 @@ module Crystal::System::File
 
     oflag = m | o
   end
+
+  # Closes the internal file descriptor without notifying libevent.
+  # This is directly used after the fork of a process to close the
+  # parent's Crystal::Signal.@@pipe reference before re initializing
+  # the event loop. In the case of a fork that will exec there is even
+  # no need to initialize the event loop at all.
+  # def file_description_close
```

Comment:
This seems to be misplaced.....

Reply:
Why? is the intended behavior of the method that should implement all the systems.

Reply:
This comment is inside a method....

Reply:
Oh, I'm blind, never mind!!